web/js/data/results.js

```diff
@@ -269,7 +283,7 @@ wv.data.results.dateTimeLabel = function (time) {
   return self;
 };
 
-wv.data.results.densify = function () {
+export function dataResultsDensify() {
```

Comment:
What's the difference between prefixing all of the data functions with `data` and prefixing all of the Worldview JavaScript files with `wv.`? Does the extra prefix provide some value? Or are we being more verbose than we need to be?